opencti-platform/opencti-front/src/private/components/settings/RulesList.jsx

```diff
@@ -549,7 +549,7 @@ const RulesListComponent = ({ relay, data, keyword }) => {
                     <Paper
                       variant="outlined"
                       classes={{ root: classes.paper }}
-                      style={{ marginTop: 25 }}
+                      style={{ marginTop: 23 }}
```

Comment:
I don't think it's the right solution for the display issue, even though it seem to work. I think the rule title should be moved outside of the grid, and we should remove completly this marginTop here.

Reply:
also it would be really nice to add a overflowX: 'auto' here so that it finally displays the whole rule definition for small screens.

Reply:
> I don't think it's the right solution for the display issue, even though it seem to work. I think the rule title should be moved outside of the grid, and we should remove completly this marginTop here.

I don't know how to make the change because the rule name is now part of the `DangerZoneBlock` component 😕